doc/experimental/kube-prometheus-stack.md

```diff
@@ -0,0 +1,244 @@
+# Migrate to kube-prometheus-stack
+
+### Version: v0.0.1-draft
+
+The purpose of this document is describe how to migrate the Verrazzano kube-prometheus-stack (named prometheus-operator)
+to the catalog kube-prometheus-stack. Once this is done, upgrades to kube-prometheus-stack can be done using the catalog. 
+This migration preserves Prometheus metrics data and does not change any component versions.
+
+This migration is needed because Verrazzano installs the kube-prometheus-stack using a Helm release named prometheus-operator.  
+As a result, all the related resources that use {RELEASE-NAME} in the Helm manifests will have the name prometheus-operator. 
+Consequently, you cannot directly upgrade to the version of kube-prometheus-stack that exists in the catalog.
+
+In addition, the image section of the overrides file must be modified to use the correct images required
+by Oracle Cloud Native Environment.
+
+***NOTE***
+These instructions assume there are 2 Prometheus replicas.  If the replica count is different, then adjust accordingly.
+
+The steps are summarized below:
+1. export the Helm user-provided overrides to an overrides file
```

Comment:
Maybe in next pass capitalize the start of each of these numbered items

Reply:
ok